### PR TITLE
get all articles without body property and comment count property kata 5

### PR DESCRIPTION
**Pull request by imaa04**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -35,14 +34,88 @@ describe("GET /api/topics", () => {
...
+  });
+});
+describe("/api/articles", () => {
+  test("GET: 200 sends the articles and they should be sorted by date in descending order with no body propert present", () => {
+    return request(app)
+    .get('/api/articles')
+    .expect(200)
+    .then((response)=> {
+      const articles = response.body.articles
+      expect(articles).toHaveLength(13)
```
Comment (johnhutcheon) [inline __tests__/app.test.js:87]: nice work checking the length before using forEach 👍

File: __tests__/app.test.js [hunk 2]
```diff
@@ -35,14 +34,88 @@ describe("GET /api/topics", () => {
...
+    return request(app)
+    .get("/api/articles")
+    .expect(200)
+    .then((response) => {
+      const articles = response.body.articles
+      expect(articles.at(0).article_id).not.toBe(1)
+      expect(articles.at(0).created_at).toBe("2020-11-03T09:12:00.000Z");
+    })
+  })
+  test("GET:404 responds with an appropriate status and error message when given a non-existent api", () => {
```
Comment (johnhutcheon) [inline __tests__/app.test.js]: This test isn't specific to the api/articles path, so can either be removed, or put into it's own describe block for incorrect paths. 👍

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
nice work Ima! I've just left a comment for amending one of your tests slightly if you could do this please. I've also noted that your controller and model files are named `topics.controllers` and `topics.models`. As you have code that relates to the articles in there too, I would rename these so that they're more accurate. They should only contain the word topics if they only relate to topics endpoints 👍

**Review by Keb84 — APPROVED** [review]
Nice refactor 🤸 merge when ready 👍

**Pull request merged**